CONTRIBUTING.md

```diff
@@ -29,8 +29,8 @@ This document is a guide for those maintaining Cucumber-Rails, and others who wo
     bundle install
     bin/install_geckodriver.sh
     bin/install_webpacker.sh
-    # Then to run tests on one specific gemfile (i.e. Rails 4.2), run
-    bundle exec appraisal rails_4_2 rake test
+    # Then to run tests on one specific gemfile (i.e. Rails 6.0), run
```

Comment:
```suggestion
    # Then to run tests on one version-specific Gemfile (e.g. gemfiles/rails_6_0.gemfile), run
```

Wording like this also shows concretely where the individual file is located. Perhaps good?

Reply:
Much much better :+1: